lib/matplotlib/tests/test_ticker.py

```diff
@@ -1449,6 +1449,23 @@ def test_latex(self, is_latex, usetex, expected):
             assert fmt.format_pct(50, 100) == expected
 
 
+@pytest.mark.xfail
```

Comment:
I'm skeptical of adding an xfail test for this, it will likely go unnoticed if future changes affect this behavior.

Unfortunately, locale is very platform/install dependent behavior that makes this harder to work around.

My primary dev system, for instance (and seemingly our CI), does not have a `fy_DE.UTF-8`.
I do have `de_DE.UTF-8`, which I think has the same behavior regarding using commas as the decimal separator.

Even if we resort to xfail if the locale raises, I'd like to see something that at least passes on CI to prevent future regressions as best we can.

This could look something like:

```python
try:
    ...
except locale.Error:
    pytest.skip("Locale ... not supported on this machine")
finally:
    locale.setlocale(locale.LC_ALL, currentLocale)
```

instead of using the `pytest.mark.xfail` decorator.

(And then looking at the CI output to make sure that at least _some_ of our CI (preferably all, but may not be easy) does _not_ skip the test.)

Reply:
I initially wanted to add a `pytest.warn` but that is being treated as a failure. I can see that `pytest.skip` is also used in `test_use_locale`. I'll update it according to your suggestion. Thanks for the review! @ksunden